### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by Tumpa-Prizrak** [SebLague/Chess-Challenge#156] [Ideas]
Why don't we create a community discord server for this challenge? This would give us the ability to find teams, test our bots against others, help with questions and issues in bot code faster than Github Discussions, etc.

**Comment by AngelFireLA** [comment]
+1

**Comment by Tumpa-Prizrak** [comment]
I'm going to create it. Think it'll be ready an about an hour. If you have something against this idea - I'll be happy to hear it <3

**Comment by Tumpa-Prizrak** [comment]
Here's the server: https://discord.gg/pAadhun2px

P. S. @SebLague, I'll be happy if you post it to readme and help me with github's webhooks

**Comment by Skoolin** [comment]
there are lots of discussions on the discord, and more then 130 users so far (after 1 day)

**Comment by Moonwalker316** [comment]
+1

**Comment by Saverio976** [comment]
Thanks for the discord, tons of tips
